Makefile

```
@@ -7,7 +7,7 @@ NAMESPACE=nobl9
 NAME=nobl9
 BIN_DIR=./bin
 BINARY=$(BIN_DIR)/terraform-provider-$(NAME)
-VERSION=0.23.0
+VERSION=0.24.0
```

```suggestion
VERSION=0.23.0
```
It seems that next version is 0.23.0 not 0.24.0